api-extractor/src/definitions/ApiDocumentation.ts

```diff
@@ -402,6 +405,14 @@ export default class ApiDocumentation {
       this.reportError('More than one API Tag was specified');
     }
 
+    if (this.apiItem instanceof ApiPackage) {
+      if (apiTagCount > 0) {
+        const tag: string = '@' + ApiTag[this.apiTag].toLowerCase();
+        this.reportError(`The ${tag} tag is not allowed on the package`);
```

Comment:
Suggest to clarify `The ${tag} tag is not allowed on the package, which is always public`